src/views/settings/SettingsMagic.tsx

```diff
@@ -29,6 +29,12 @@ const SettingsMagic: Screen<"SettingsMagic"> = ({ navigation }) => {
         <NativeItem
           trailing={
             <Switch
+              trackColor={
+                {
+                  false: colors.border, true: colors.primary
```

Comment:
Juste une p'tite amélioration de l'indentation :

```suggestion
                  false: colors.border,
                  true: colors.primary
```